From 8d78bce70bf5501e6b56ba488ab746f128ba59b2 Mon Sep 17 00:00:00 2001
From: albert-github <albert.tests@gmail.com>
Date: Mon, 5 Jun 2023 10:25:07 +0200
Subject: [PATCH] issue #10110 Regression: Specified markdown mainpage '...md'
 has not been defined as input file

The filename to be used should be the full qualified filename / filename with absolute path as otherwise the file is not found.
---
 src/doxygen.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/doxygen.cpp b/src/doxygen.cpp
index ed9e324b18..0b54dae059 100644
--- a/src/doxygen.cpp
+++ b/src/doxygen.cpp
@@ -11712,7 +11712,7 @@ static void checkMarkdownMainfile()
       return;
     }
     bool ambig = false;
-    if (findFileDef(Doxygen::inputNameLinkedMap,mdfileAsMainPage,ambig)==0)
+    if (findFileDef(Doxygen::inputNameLinkedMap,fi.absFilePath(),ambig)==0)
     {
       warn_uncond("Specified markdown mainpage '%s' has not been defined as input file\n",qPrint(mdfileAsMainPage));
       return;
